bin/Powheg/production/2017/13TeV/ggHH_EWChL/powheg_cHHH0.input

```diff
@@ -0,0 +1,118 @@
+! ggHH production parameters:
+mtdep 3              ! 0: Higgs effective field theory (HEFT)
+!                      1: Born improved HEFT
+!                      2: approximated full theory (FTapprox)
+!                      3: full theory
+
+hmass 125            ! Higgs boson mass
+topmass 173          ! top quark mass (THIS VALUE IS HARD CODED IN THE VIRTUAL
+                     ! MATRIX ELEMENT AND FOR CONSISTENCY HAS NOT TO BE CHANGED WHEN
+                     ! RUNNING FULL THEORY PREDICTIONS - i.e. mtdep=3)
+hdecaymode -1        ! PDG code for Higgs boson decay products (it affects only the SMC)
+!                      allowed values are:  
+!                      0 all decay channels open
+!                      1-6 d dbar, u ubar,..., t tbar (as in HERWIG)
+!                      7-9 e+ e-, mu+ mu-, tau+ tau-
+!                      10  W+W-
+!                      11  ZZ
+!                      12  gamma gamma      
+!                      -1  all decay channels closed
+cHHH 0.0
+
+! general parameters:
+numevts NEVENTS
+ih1         1        ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2         1        ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500d0        ! energy of beam 1
+ebeam2 6500d0        ! energy of beam 2
+! To be set only if using LHA pdfs
+lhans1 325300         ! pdf set for hadron 1 (LHA numbering)
+lhans2 325300	      ! pdf set for hadron 2 (LHA numbering)
+! To be set only if using internal mlm pdf
+# ndns1 131          ! pdf set for hadron 1 (mlm numbering)
+# ndns2 131          ! pdf set for hadron 2 (mlm numbering)
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5 0.25    ! for not equal pdf sets 
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1    ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1    ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+!
+ncall1  10000        ! number of calls for initializing the integration grid
+itmx1       3        ! number of iterations for initializing the integration grid
+ncall2  10000        ! number of calls for computing the integral and finding upper bound
+itmx2       3        ! number of iterations for computing the integral and finding upper bound
+foldcsi     1        ! number of folds on csi integration
+foldy       1        ! number of folds on  y  integration
+foldphi     1        ! number of folds on phi integration
+nubound 10000        ! number of bbarra calls to setup norm of upper bounding function
```

Comment:
did you test that this is sufficient? what is the fraction of neg. weights?

Reply:
The negative weight fraction for cHHH1 was 5.17e-2. Please let me know if that is low enough and if not, what fraction I should be aiming for.